.github/workflows/multi-gitter-pr-sync.yml

```diff
@@ -0,0 +1,53 @@
+name: Creating PR using Multi-Gitter
+
+on:
+  pull_request:
+    types: [opened]
+    branches:
+      - develop
+      - main
+
+jobs:
+  update-dependencies:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: '>=1.17.0'
+
+      - name: Install multi-gitter
+        run: curl -s https://raw.githubusercontent.com/lindell/multi-gitter/master/install.sh | sh
```

Comment:
This looks unsafe, can we fork this and use it? Or use a commit pinned version? @YashasviDevtron

Reply:
Have updated